test/p2ptest/js/test.js

```diff
@@ -250,21 +251,10 @@ TestClient.prototype = {
     return result
   },
 
-  showInPage: function(stream) {
-    var that = this;
-    var video = document.createElement("video"),
-    videoId = "stream" + stream.id;
-    video.setAttribute("id", videoId);
-    video.setAttribute("width", "320px");
-    video.setAttribute("height", "240px");
```

Comment:
why remove this showInPage to test-ci.js ?  how to share this with nightly test cases ?